lib/core/Site.js

```diff
@@ -159,6 +159,9 @@ class Site extends React.Component {
                 }}
               />
             ))}
+          {process.env.NODE_ENV !== 'production' && (
+            <script src="http://localhost:35729/livereload.js" />
```

Comment:
This port number needs to be consistent with the server configuration. Hence extract the port out into a constant.

Reply:
👍thanks, this is a good improvement, makes sense